qemu/tests/blockdev_inc_backup_mod_readonly_bitmap.py

```diff
@@ -0,0 +1,64 @@
+from virttest.qemu_monitor import get_monitor_function
+from virttest.qemu_monitor import QMPCmdError
+
+from provider.block_dirty_bitmap import block_dirty_bitmap_add
+from provider.blockdev_live_backup_base import BlockdevLiveBackupBaseTest
+
+
+class BlockdevIncbkModRdonlyBitmapTest(BlockdevLiveBackupBaseTest):
+    """Clear/Remove readonly bitmaps"""
+
+    def prepare_test(self):
+        self.preprocess_data_disks()
+        self.prepare_main_vm()
+        self.prepare_data_disks()
+
+    def modify_readonly_bitmaps(self):
+        for act in ['block-dirty-bitmap-clear', 'block-dirty-bitmap-remove']:
+            f = get_monitor_function(self.main_vm, act)
+            try:
+                f(self._source_nodes[0], self._bitmaps[0])
+            except QMPCmdError as e:
+                error_msg = self.params['error_msg'].format(
+                    bitmap=self._bitmaps[0])
+                if error_msg not in str(e):
+                    self.test.fail('Unexpected error: %s' % str(e))
+            else:
+                self.test.fail('%s succeeded unexpectedly' % act)
+
+    def add_persistent_bitmap(self):
+        kargs = {'bitmap_name': self._bitmaps[0],
+                 'target_device': self._source_nodes[0],
+                 'persistent': 'on'}
+        block_dirty_bitmap_add(self.main_vm, kargs)
+
+    def restart_vm_with_readonly_data_image(self):
+        self.main_vm.monitor.system_powerdown()
```

Comment:
Can we add "try...except" to catch some error here since I met once qemu coredump for the test?

Reply:
Nini, do you still remember which exception was thrown, QMPCmdError, QMPEventError or any other one? 
also any exception can break the test run, even though we never catch it.

Reply:
It's a qemu core dump, the case continue to execute and PASS, it's weird.

Reply:
vm.destroy() caused the crash, no exception, and all qmp commands returned successfully.
so add a core dump check in the end of do_test, in case we hit another crash in future.